examples/artifact-path-placeholders.yaml

```diff
@@ -0,0 +1,36 @@
+# This example demonstrates the how to refer to input and output artifact paths.
+# Referring to the path instead of copy/pasting it prevents errors when paths change.
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: artifact-path-placeholders-
+spec:
+  entrypoint: grep
+  arguments:
+    parameters:
+    - name: regex
+      value: Argo
+    artifacts:
+    - name: text
+      raw:
+        data: |
```

Comment:
Should we limit the data size?

Reply:
Are asking me to reduce the size of this particular sample?

Reply:
Done